app/src/main/java/com/hedvig/app/SplashActivity.kt

```diff
@@ -37,11 +36,10 @@ class SplashActivity : BaseActivity(R.layout.activity_splash) {
     override fun onStart() {
         super.onStart()
 
-        disposables += loggedInService
-            .getLoginStatus()
-            .subscribeOn(Schedulers.io())
-            .observeOn(AndroidSchedulers.mainThread())
-            .subscribe({ navigateToActivity(it) }, { e(it) })
+        CoroutineScope(IO).launch {
```

Comment:
This can't be correct, can it?

Reply:
it works so i think so

Reply:
This coroutine needs be scoped to the calling activity (so that it's cancelled when the activity goes out of scope).